src/vaults/vaults.utils.ts

```diff
@@ -55,6 +51,7 @@ export async function defaultVault(
   const bouncer = vaultDefinition.bouncer ?? BouncerType.None;
   const type = vaultDefinition.protocol === Protocol.Badger ? VaultType.Native : VaultType.Standard;
   const behavior = vaultDefinition.behavior ?? VaultBehavior.None;
+  const version = vaultDefinition.version ?? VaultVersion.v1;
```

Comment:
default is to not use the last field, and now its on the def so use it here instead